code/game/atoms.dm

```diff
@@ -31,7 +31,12 @@
 	var/default_alpha = 255
 	var/default_scale = 1
 
-
+	///overlays that should remain on top and not normally removed when using cut_overlay functions, like c4.
+	var/list/priority_overlays
+	///a very temporary list of overlays to remove
+	var/list/remove_overlays
+	///a very temporary list of overlays to add
+	var/list/add_overlays
```

Comment:
This is not a good idea, adding three new lists to every atom in the game.

How much of this is ported? I seem to recall the PRs you showed me only added one list, not three

Couldn't this data be stored in an extension for the things that need it, instead of new vars?